connector/tableau/looker-jdbc/dialect.tdd

```diff
@@ -667,7 +667,8 @@
       <argument type='spatial' />
     </function>
     <function group='logical' name='IIF' return-type='bool'>
-      <formula>IF((%1), (%2), (%3))</formula>
+      <!-- To prevent IF from coercing NULL to FALSE, wrap in another IF that checks for NULL. -->
+      <formula>IF((%1) IS NOT NULL, IF((%1), (%2), (%3)), NULL)</formula>
```

Comment:
Were you able to test this with `val3` supplied? 
I'm seeing 4 values in (bool, val1, val2, val3), but in <formula> there are only 3 (%1, %2, %3)  so I'm having a hard time lining them up

Reply:
Yeah all of the ones with val3 supplied (see the formulas above or below) already did this

Reply:
basically for each type, theres two formulas: one with val3 supplied, and one without. All the formulas with val3 supplied already had this wrapper (it's how i got this idea), i just fixed the ones WITHOUT val3.

does that answer your question ?

Reply:
Yeah, I didnt see those earlier. thanks